Mu2eKinKal/src/ANNStrawHitUpdater.cc

```diff
@@ -0,0 +1,71 @@
+#include "Offline/Mu2eKinKal/inc/ANNStrawHitUpdater.hh"
+#include "Offline/TrackerConditions/inc/StrawResponse.hh"
+#include "Offline/TrackerGeom/inc/Straw.hh"
+#include "Offline/RecoDataProducts/inc/ComboHit.hh"
+#include <cmath>
+
+namespace mu2e {
+  using KinKal::ClosestApproachData;
+  using KinKal::VEC3;
+  ANNStrawHitUpdater::ANNStrawHitUpdater(ANNSHUConfig const& annshutuple) {
+    mva_  = new MVATools(std::get<0>(annshutuple));
```

Comment:
This looks like a memory leak.  My preference would be to unique_ptr

Reply:
correct, a leak, I fixed it.  Sady, unique_ptr is incompatible with std::any, which is used to store objects of this class in the config.  see https://stackoverflow.com/questions/66353692/stdany-of-stdunique-ptr-doesnt-work